client/src/js/ResultsTable.js

```diff
@@ -0,0 +1,82 @@
+import React, { Component } from 'react';
+import '../css/App.css';
+
+
+class Result {
+    constructor(title, description, excerpt, location, postUrl, applyUrl) {
+    this.title = title;
+    this.description = description;
+    this.excerpt = excerpt;
+    this.location = location;
+    this.postUrl = postUrl;
+    this.applyUrl = applyUrl;
+    this.selected = false;
+    this.i = 0;
+  }
+}
+
+class ResultsTable extends Component {
+  constructor(props) {
+      super(props);
+      this.state = {
+        results: props.results,
+      }
+  }
+
+  handleClick(action, result) {
+    switch (action) {
+      case 0: // Clicked on row to select/deselect
```

Comment:
Javascript switch statements can switch on strings so you could use a descriptive string for the action instead of just 0, 1, and 2. Using numbers can get confusing.

Reply:
Replaced 0, 1, 2 with "select", "post", and "apply"